pkg/cli/cmd/templates.go

```diff
@@ -43,16 +45,106 @@ opa:
 `
 
 const templatePreamble = `---
+# compatible config version
+version: 1 
+
 logging:
   prod: true
   log_level: info
 
 directory:
   db_path: ${TOPAZ_DIR}/db/directory.db
   seed_metadata: {{ .SeedMetadata }}
-  
+
+# remote directory is used to resolve the identity for the authorizer.
+remote_directory:
+  address: "0.0.0.0:9292" # set as default, it should be the same as the reader as we resolve the identity from the local directory service.
+  insecure: true
+
+# default jwt validation configuration
```

Comment:
I would suggest instead of commenting it all out, only adding a comment behind the `acceptable_time_skew_seconds` value like:
```
# jwt validation configuration
jwt:
  acceptable_time_skew_seconds: 5 # set as default, 5 secs
```